### PR TITLE
Add Replay button when player reaches end of content

### PR DESCRIPTION
**Pull request by mrocajr**
## Description
Added a replay button that appears when the player triggers 'ended'

## Specific Changes proposed
- Add handleEnded() function to play-toggle
- Added vjs-ended to vjs-play-control

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by gkatsev** [comment]
Can you rebase this to remove the first commit?

**Comment by mrocajr** [comment]
Sure

File: src/js/control-bar/play-toggle.js [hunk 1]
```diff
@@ -68,6 +70,18 @@ class PlayToggle extends Button {
...
+  /**
+   * Add the vjs-ended class to the element so it can change appearance
+   *
+   * @method handleEnded
+   */
+  handleEnded() {
+    this.removeClass('vjs-playing');
+    this.addClass('vjs-ended');
+    // change the button text to "Replay"
+    this.controlText('Replay');
```
Comment (misteroneill) [inline src/js/control-bar/play-toggle.js]: "Replay" isn't in our translations so we should make sure it's in `en.json` and run `grunt check-translations` to make sure it gets added to the needed translations document.

File: src/css/components/_play-pause.scss [hunk 2]
```diff
@@ -6,3 +6,6 @@
 .video-js .vjs-play-control.vjs-playing {
   @extend .vjs-icon-pause;
 }
+.video-js .vjs-play-control.vjs-ended {
+  @extend .vjs-icon-replay
```
Comment (misteroneill) [inline src/css/components/_play-pause.scss]: Definitely not critical at all, but we should probably add a semicolon here.

**Comment by mrocajr** [comment]
Changes based on CR

**Comment by misteroneill** [comment]
Don't forget to run `grunt check-translations` so we pick up "Replay" 😄

**Comment by mrocajr** [comment]
Yep, I ran it here and checked this in: https://github.com/mrocajr/video.js/blob/d2c57642e4787e142297aaa80556ae55519bf98d/docs/translations-needed.md

**Comment by gkatsev** [comment]
@misteroneill github recently started hiding some diffs, so, it's possible you missed it because it's collapsed. It was for me.

**Comment by misteroneill** [comment]
Weird, OK. 👍

**Comment by BrandonOCasey** [comment]
@mrocajr looks like this needs a rebase

**Comment by mrocajr** [comment]
To avoid further rebase chaos, I've put a new PR here: https://github.com/videojs/video.js/pull/3868

**Pull request closed without merging**